check-pr-description/README.md

```diff
@@ -24,26 +24,35 @@ The supported components are:
 - `renku-ui` (that includes the `renku-ui-server` component, since the version is
   aligned between the 2 components)
 
-The reference will be stored in an output variable with the same name.
+The reference will be stored in an output variable with the same name. For example,
 
-The `#notest` string will falsify the otherwise truthy variable named `test-enabled`.
+```
+/deploy renku-ui=0.11.9 renku=master
+```
 
-Keep in mind that:
-- Unknown sections are skipped.
-- The order plays no role unless you define multiple references for the same
-  component. In that case, only the first is considered. It would be best if you
-  tried to avoid any repetition.
+will deploy the tag `0.11.9` of `renku-ui` and the tip of the `master` branch of `renku`.
+### Passing in additional values
 
-## How to use that
+You may pass in additional ad-hoc values to the deployment by using the `extra-values` option.
+For example:
 
-This action is meant to be used in combination with the other actions from this
-repository to simplify deploying a testing environment for pull requests.
+```
+/deploy --extra-values="tests.image.ref=my-test,core.sentry.env=feature"
```

Comment:
I think it's just
```suggestion
/deploy extra-values="tests.image.ref=my-test,core.sentry.env=feature"
```
(see https://github.com/SwissDataScienceCenter/renku-actions/blob/master/check-pr-description/action.yaml#L87)

Reply:
thanks you're right